From 9178350ddac414c7899c21ad5f0afcb49219e8f1 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 12 Jun 2015 11:22:59 -0400
Subject: [PATCH 1/3] Add pycrypto to setup.py

The change from https://reviews.reviewboard.org/r/7111/ resulted in
PyCrypto being required for all deployments of ReviewBoard.
Previously, it would have only been required if certain SCMs were in
use (beanstalk, bitbucket, gitlab or unfuddle).

However, it is now imported and used in the server initialization,
which means we need to ensure that it is on the system.

Yes, most systems will have pycrypto available already, but it's
always better to be explicit and avoid unexpected failures.
---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index a39f78a783622adef1fce3853809241b5875a9c2..d9ed3e227258248e1c4b26b5dd2bc7e338041826 100755
--- a/setup.py
+++ b/setup.py
@@ -194,10 +194,11 @@ setup(name=PACKAGE_NAME,
           ],
       },
       cmdclass=cmdclasses,
       install_requires=[
           django_version,
+          'pycrypto',
           'django_evolution>=0.7.5,<=0.7.999',
           'django-haystack>=2.3.1',
           'Djblets>=0.8.19,<=0.8.999',
           'docutils',
           'markdown>=2.4.0,<2.4.999',
-- 
2.4.3

